Classes/EventPublisher/JobQueueEventPublisher.php

```diff
@@ -0,0 +1,84 @@
+<?php
+declare(strict_types=1);
+namespace Neos\EventSourcing\EventPublisher;
+
+/*
+ * This file is part of the Neos.EventSourcing package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Flowpack\JobQueue\Common\Job\JobManager;
+use Neos\EventSourcing\Event\DecoratedEvent;
+use Neos\EventSourcing\Event\DomainEvents;
+use Neos\EventSourcing\EventListener\Mapping\EventToListenerMapping;
+use Neos\EventSourcing\EventListener\Mapping\EventToListenerMappings;
+use Neos\EventSourcing\EventPublisher\JobQueue\CatchUpEventListenerJob;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * An Event Publisher that sends events to a Job Queue using the Flowpack.JobQueue package.
```

Comment:
It doesn't really send the events, but just a notification per listener that it needs to update. I think that's an important distinction and it would maybe even make sense to explain in short, why this is done (to not put the burden of deduplicating the events in the queue on the listener, so that it can just track the position and happily live his exactly-once-processing life)

Reply:
Good call, I tried to make this more clear: e39b269f0fb0c403c3a18972c2be4f1a67a532f7

BTW: More documentation is coming asap!